src/API.jl

```diff
@@ -1036,7 +1042,7 @@ function instantiate(ctx::Context; manifest::Union{Bool, Nothing}=nothing,
     Operations.download_artifacts(ctx, [dirname(ctx.env.manifest_file)]; platform=platform, verbose=verbose)
     # check if all source code and artifacts are downloaded to exit early
     if Operations.is_instantiated(ctx) 
-        _do_auto_precompile() && Pkg.precompile()
+        _do_auto_precompile() && Pkg.precompile(true)
```

Comment:
The call to `precompile` could be moved in to the `_do_auto_precompile` function (I know this isn't from this PR but just something I noted).

Reply:
Done